### ISSUE TITLE
Questions about Deeplab retrain part

### ISSUE THREAD

**Issue by lh-zhu** [qjadud1994/DRS#4]
Greetings！

I read your paper again and again recently, and I really learned a lot from your work. For the retrain part, I met some problems, would you like to help to answer?

At first, I noticed that your results in Table.4 were devided into 3 parts, the subtitle of first part is  **Segmentation Network : DeepLab-Large-FOV (VGG-16)**, the 2st part is **Segmentation Network : DeepLab-Large-FOV (ResNet-101)**, the 3rd part is **Segmentation Network : DeepLab-ASPP (ResNet-101)**.

My first question is that your **deeplab retrained script**(`/DRS/DeepLab-V2-PyTorch/train.sh`) indicate which part in Table.4 we mentioned before? What about other two parts' code? I could not find a VGG-Backboned model in `DeepLab-V2-PyTorch` folder.

**Comment by stickyfiner** [comment]
@qjadud1994 Hi, I retrained DeepLabV2 with your shared refined pseudo segmentation labels. But the performance only achieves 64.8% and 65.7(after crf). All hyper-parameters follow your provided code. Could you please point out the possible mistake? Thank you very much!

**Comment by stickyfiner** [comment]
@qjadud1994 Hi, I retrained DeepLabV2 with your shared refined pseudo segmentation labels. But the performance only achieves 64.8% and 65.7(after crf). All hyper-parameters follow your provided code. Could you please point out the possible mistake? Thank you very much!

**Comment by Unrealluver** [comment]
> Thank you for your interest in our work.
> 
> For your first question, DeepLab Large-FOV is equal to DeepLab-V1, and DeepLab-ASPP is equal to DeepLab-V2.
> The DeepLab-V2 code produces DeepLab-ASPP (ResNet-101) performance.
> 
> Since most of the recent works adopt DeepLab-V2 or V3, which shows higher performance than DeepLab-V1, we upload only the DeepLab-V2 (DeepLab-ASPP) code.
> 
> If you need the DeepLab-V1 (DeepLab Large-FOV (VGG-16 or ResNet-101) code, we will upload that soon.

Thank you for your early replay, looking for your additional code!
